app/models/user.rb

```diff
@@ -264,6 +264,17 @@ def block!
     end
   end
 
+  def unblock!
+    update!(
```

Comment:
Raise if not blocked?

Reply:
We can no-op if the user already is in the correct state